src/app/home/avatar-generator/avatar-generator.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="center-align container">
+  <div class="mx-1 my-2">
+    <h4 class="title pixel-text">Crea tu propio avatar</h4>
+  </div>
+  <div class="row">
+    <div class="col s12 m6">
+      <div #screen class="bg-image center">
+        <img class="logo" src="assets/images/logo.svg" alt="cielo de noche">
+        <div [innerHTML]="buildAvatarSvg() | sanitizeHtml"></div>
```

Comment:
This method is getting called a lot of times even when there are no changes in the avatar (clicking anywhere makes at least 4 calls to this method). It would be good to optimize this to avoid calling it many times. Two possible approaches could be:
1. Making this component use the `OnPush` change detection strategy.
2. Building the avatar svg just when it's necessary and storing the result in a variable, then the template can reference it instead of calling the method.

Reply:
fixed

Reply:
The fix was added to the attribute selector component but not to this one, please also fix it here :)